test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -225,9 +225,8 @@ proc ::AlgorandGoal::StartNetwork { NETWORK_NAME NETWORK_TEMPLATE TEST_ALGO_DIR
         spawn goal network status -r $TEST_ROOT_DIR
         expect {
             timeout { close; ::AlgorandGoal::Abort "Timed out retrieving network status"  }
-            ".*Error getting status.*" { close; ::AlgorandGoal::Abort "error getting network status: $expect_out(buffer)"}
-            "^Network Started under.*"   { puts "Network $NETWORK_NAME status ok"; close }
```

Comment:
Do you want to retain this line?  It shows that the network started ok and ok to continue.

Reply:
there is nothing to retain here. the `goal network status` never prints out `Network Started under*`.

Reply:
OK